collectors/charts.d.plugin/sensors/README.md

```diff
@@ -1,4 +1,4 @@
-# sensors
+# Sensors monitoring with Netdata
```

Comment:
I think we could be more specific here and use `Linux Machines Sensors` (modules monitors only linux machine sensors)

Reply:
**Network UPS tools monitoring with Netdata**

Reply:
**Linux machine sensors monitoring with Netdata**